### DISCUSSION TITLE
SignInWithPassword invalid credentials issue

### DISCUSSION THREAD

**Discussion by nimeshmaharjan1** [supabase/supabase#11885] [Questions]
Getting invalid credentails error even when I am sending the correct password and email is there any thing I need to do or decrypt the password before calling the supabase.auth.signInWithPassword method?

![image](https://user-images.githubusercontent.com/86953478/214221166-3fb5425a-23cf-4c07-9c4e-57f4978ee1a4.png)

I have tried the magic link its works but only facing issue with the password sign in

![image](https://user-images.githubusercontent.com/86953478/214221273-3e4c62c9-0527-459c-84e6-51fd4ae0ef34.png)

![image](https://user-images.githubusercontent.com/86953478/214221388-bd3ce158-769c-41b7-92b6-033539696ff9.png)

![image](https://user-images.githubusercontent.com/86953478/214221481-7f035237-a60b-4f3b-bcf7-911a3659e8dc.png)

![image](https://user-images.githubusercontent.com/86953478/214221504-e5c753af-0e1b-441d-9cfa-191ba894c603.png)

**Comment by GaryAustin1** [comment]
This could be your anon_key is not set correctly in createClient.  Please check the apikey and authorization header look correct with no extra characters.

Edit, although if magic link works....
Also you do not need to encode or do anything special with the password, other than get it correct.
